devlib/target.py

```diff
@@ -3009,17 +3012,17 @@ def __init__(self,
             if key in ssh_conn_params
         )
 
-        super(ChromeOsTarget, self).__init__(connection_settings=self.ssh_connection_settings,
-                                             platform=platform,
-                                             working_directory=working_directory,
-                                             executables_directory=executables_directory,
-                                             connect=False,
-                                             modules=modules,
-                                             load_default_modules=load_default_modules,
-                                             shell_prompt=shell_prompt,
-                                             conn_cls=SshConnection,
-                                             is_container=is_container,
-                                             max_async=max_async)
+        super().__init__(connection_settings=self.ssh_connection_settings,
+                         platform=platform,
+                         working_directory=working_directory,
+                         executables_directory=executables_directory,
+                         connect=False,
+                         modules=modules,
+                         load_default_modules=load_default_modules,
+                         shell_prompt=shell_prompt,
+                         conn_cls=SshConnection,
+                         is_container=is_container,
+                         max_async=max_async)
```

Comment:
might go the extra mile and just remove the link between length of the function name and indentation:
```
foobarextremelylongandalsowewillchangeittomorrowandbreakindentation(
    iamaparameterandidontcareaboutmyfunctionnamelength=42,
    ...,
)
```